book/content/citing_cryocloud.md

```diff
@@ -0,0 +1,6 @@
+
+# Citing CryoCloud
+
+To track the impact that CryoCloud has on publications and research output, 
+we ask that any projects and publications that the CryoCloud supports use 
+our doi: [10.5281/zenodo.7389522](https://doi.org/10.5281/zenodo.7389522).  
```

Comment:
The title on the Zenodo page still says 'ICESat-2 Hackweek Website', might need to edit it manually. I think it might be automatically pulling the metadata from CITATION.cff (which probably needs updating too).